Tasksets/sets.md

```diff
@@ -0,0 +1,9 @@
+# Sets of tasks
+
+## A simple task set
+
+This set is just a simple set that comports 3 tasks. It is mainly an example of the task export system.
+
+```lua
+W3sidGFzayI6IkEgc2ltcGxlIHRhc2siLCJ0aXRsZSI6IlRhc2sgMSJ9LHsidGFzayI6IkFuIG90aGVyIHNpbXBsZSB0YXNrIiwidGl0bGUiOiJUYXNrIDIifSx7InRhc2siOiJZZXQgYW4gb3RoZXIgdmVyeSBzaW1wbGUgdGFzayIsInRpdGxlIjoiVGFzayAzIn1d
+```
```

Comment:
The string is different from the other PR :D

Reply:
It is different as I had to now save the title and the task. More info to stock.